dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -169,6 +168,30 @@ internal constructor(internal var handler: LogHandler) {
         internalLog(priority, message, throwable, attributes)
     }
 
+    /**
+     * Sends a log message with strings for error information.
```

Comment:
I'm wondering if we should add the note here that this method is not supposed to be used in native apps directly. My motivation is that is this method is misused in the native apps and stack trace is not a valid JVM stacktrace, then we will have a growth of the de-obfuscation call errors, which may be confusing for the infra monitoring. WDYT?

But anyway we don't have this for the `RumMonitor` counterpart.

Reply:
Yes, I'll add a note to prefer the other methods.

We do / should have a counterpart in RumMonitor? We have a way to send a string stackTrace anyway....

Reply:
Yes, I mean we don't have the sentence saying that it is mostly for the internal usage in case of `RumMonitor`'s counterpart.

Reply:
Ah, I'll add it in both places.